tests/integration/test_paths_simple.py

```diff
@@ -113,17 +113,17 @@ def generate_topology():
 
         links["S1:1<->S2:1"] = Link(interfaces["S1:1"], interfaces["S2:1"])
         links["S1:1<->S2:1"].extend_metadata(
-            {"bandwidth": 50, "ownership": "red"}
```

Comment:
Unit test should still pass with link metadata using `{"ownership": "str"}` value since we do have prod usage using this format, so let's ensure unit test cov and still not break compat.

Reply:
Undid this change, tests now back to there original state